### PR TITLE
Fix ranger multi-advertise #2480

### LINKED ISSUE #2480: DISTANCE_SENSOR multi-subscribe still broken
I've been trying to get the PX4Flow sonar + LL40LS working together. 

```ls /dev/``` gives me :
```
 range_finder0
 range_finder1
```
I can see that the PX4flow sonar gets instance ID 0 ( Can see data in QGC )
I assume that the ll40ls is getting uorb instance ID 1, but using orb_subscribe_multi() with the instance set to 1 doesn't give me any data.

Any insights?

### PR DESCRIPTION
**Pull request by mhkabir**
This finally fixes #2480

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Thanks! Could you reply to the comment I made inline?

File: src/drivers/ll40ls/LidarLiteI2C.cpp [hunk 1]
```diff
@@ -126,7 +126,7 @@ int LidarLiteI2C::init()
 
 	_class_instance = register_class_devname(RANGE_FINDER_BASE_DEVICE_PATH);
 
-	if (_class_instance == CLASS_DEVICE_PRIMARY) {
+	if (_class_instance >= 0) {
```
Comment (LorenzMeier) [inline src/drivers/ll40ls/LidarLiteI2C.cpp]: I can see that this was needed, to fix it, but now that multi orb advertisements and class instances are completely decoupled the publication should not depend on the class instance at all.

File: src/drivers/ll40ls/LidarLitePWM.cpp [hunk 2]
```diff
@@ -108,13 +108,13 @@ int LidarLitePWM::init()
 
 	_class_instance = register_class_devname(RANGE_FINDER_BASE_DEVICE_PATH);
 
-	if (_class_instance == CLASS_DEVICE_PRIMARY) {
```
Comment (LorenzMeier) [inline src/drivers/ll40ls/LidarLitePWM.cpp]: There is no need to guard this at all with the class instance.

File: src/drivers/ll40ls/LidarLiteI2C.cpp (hunk 1 above)
Comment (mhkabir) [inline src/drivers/ll40ls/LidarLiteI2C.cpp]: That's what I was thinking as well. Will remove.

**Comment by mhkabir** [comment]
@LorenzMeier Should I remove all the guards? I don't really see a point keeping them.

**Comment by LorenzMeier** [comment]
Yes, removing all of them makes sense. Note the failing CI test.

**Comment by mhkabir** [comment]
@LorenzMeier It isn't immediately apparent where the CI is failing. Could you please have a look?

**Comment by LorenzMeier** [comment]
It is immediately apparent where its failing:
1) Go to the status
2) Display the raw log
3) Search for (CTRL-F) "error 1" (without quotes)
4) Profit!

Here is the log for this run at your convenience:
https://s3.amazonaws.com/archive.travis-ci.org/jobs/84677073/log.txt

File: src/drivers/ll40ls/LidarLitePWM.cpp [hunk 3]
```diff
@@ -108,19 +108,17 @@ int LidarLitePWM::init()
...
+						     &_orb_class_instance, ORB_PRIO_DEFAULT);
 
-		if (_distance_sensor_topic == nullptr) {
-			DEVICE_DEBUG("failed to create distance_sensor object. Did you start uOrb?");
-		}
+	if (_distance_sensor_topic == nullptr) {
+		DEVICE_DEBUG("failed to create distance_sensor object. Did you start uOrb?");
 	}
-
+
```
Comment (LorenzMeier) [inline src/drivers/ll40ls/LidarLitePWM.cpp]: Whitespace

**Comment by LorenzMeier** [comment]
Could you please also provide test results for PX4FLOW only and PX4FLOW + LL40LS connected and ensure they work as expected?

**Comment by mhkabir** [comment]
Closing in favour of #3085 due to broken rebase issues.

**Pull request closed without merging**